tfhe/src/high_level_api/integers/unsigned/base.rs

```diff
@@ -619,6 +619,44 @@ where
             }
         })
     }
+
+    /// Reverse the bit of the unsigned integer
+    ///
+    /// # Example
+    ///
+    /// ```rust
+    /// use tfhe::prelude::*;
+    /// use tfhe::{
+    ///     generate_keys, set_server_key, ConfigBuilder, FheBool, FheUint16, FheUint8, MatchValues,
+    /// };
```

Comment:
No need for the MatchValues import

Also this should be implemented for `FheInt` now

Reply:
Done